kore/test/Test/Kore/Step/Simplification/Predicate.hs

```diff
@@ -304,13 +304,14 @@ simpleEvaluator
         )
     => [(TermLike variable, TermLike variable)]
     -> TermLike variable
+    -> Predicate variable
```

Comment:
Do you need the extra argument here?

Reply:
`simpleEvaluator` is used to create a `BuiltinAndAxiomSimplifier`